tests/functional/pv/pvc_snapshot/test_volume_group_snapshot.py

```
@@ -0,0 +1,264 @@
+        "restore_yaml": constants.CSI_CEPHFS_PVC_RESTORE_YAML,
+        "access_mode": constants.ACCESS_MODE_RWO,
+    },
+}
+
+PVC_COUNT = 3
+PVC_SIZE = 5
+VGS_LABEL_KEY = "app"
+VGS_LABEL_VALUE = "vgs-test"
+
+
+@green_squad
```

Ruff (0.16.3)

[error] Undefined name `green_squad`

(F821)